CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 ## [v0.11.1.0](https://github.com/LambdaHack/LambdaHack/compare/v0.11.0.0...v0.11.1.0)
 
 - Split the long Content.ModeKind texts into lines, [#283](https://github.com/LambdaHack/LambdaHack/issues/283)
+- values from UnitTestHelpers where appropriate, [#286](https://github.com/LambdaHack/LambdaHack/issues/286)
```

Comment:
Is there a start of the sentence missing?

Reply:
yes, fixed now.thnx